.github/workflows/test.yml

```diff
@@ -102,10 +102,29 @@ jobs:
       - name: Build design tokens 🛠️
         run: yarn build:tokens
 
-      - name: Lint Commit Messages 👕
-        uses: wagoid/commitlint-github-action@v5
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Lint Javascript + SCSS 👕
         run: yarn run lint --max-warnings=0
+
+  commitlint:
+    name: Lint Commit Messages 👕
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup node 😻
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.node-version'
+          cache: 'yarn'
+      - name: Install Dependencies 🧶
+        run: yarn install --immutable
+      - name: Validate commits commitlint
+        # This workflow can also be triggered via "workflow_call".
+        # Since it's a push event we have access to these properties https://docs.github.com/en/webhooks/webhook-events-and-payloads#push
+        if: github.event_name == 'push'
+        run: |
+          COMMIT_COUNT=$(echo '${{ toJSON(github.event.commits) }}' | jq length)
```

Comment:
How/where does `jq` get installed? Mostly out of curiousity...

Reply:
Great question! Github runners (the github hosted virtual machines we run workflows on) come with a set of preinstalled software. This can be seen looking at the logs of the github action. Here's the link in the image below for reference https://github.com/actions/runner-images/blob/ubuntu22/20240204.1/images/ubuntu/Ubuntu2204-Readme.md


![Screenshot 2024-02-09 at 3 12 15 PM](https://github.com/chanzuckerberg/edu-design-system/assets/94476616/df9f1493-c8a9-47cb-88c1-cc8b8cbfe3bb)